src/sources/aws_sqs/config.rs

```diff
@@ -59,6 +68,8 @@ impl SourceConfig for AwsSqsConfig {
                 decoder,
                 poll_secs: self.poll_secs,
                 concurrency: self.client_concurrency,
+                visibility_timeout_secs: self.visibility_timeout_secs,
```

Comment:
Would it make sense to do the conversion here instead of at use?

Reply:
Ah, yes, good call.